google/cloud/logging/CMakeLists.txt

```diff
@@ -17,7 +17,6 @@
 set(DOXYGEN_PROJECT_NAME "Google Cloud Logging C++ Client")
 set(DOXYGEN_PROJECT_BRIEF "A C++ Client Library for Google Cloud Logging")
 set(DOXYGEN_PROJECT_NUMBER "${PROJECT_VERSION} (Beta)")
-set(DOXYGEN_EXCLUDE_PATTERNS "*/*_test.cc")
 set(DOXYGEN_EXCLUDE_SYMBOLS "internal" "logging_internal" "logging_testing"
                             "examples")
```

Comment:
logging has no `examples`

Reply:
I know.  I am trying to make all the CMake files look as-if they had been generated, or closer to it.